content/en/docs/external-add-ons/kserve/introduction.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Introduction"
+description = "A brief introduction to KServe"
```

Comment:
We need KServe folks to review it.
/assign @yuzisun @terrytangyuan @sivanantha321